.changeset/brown-onions-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'myst-cli': patch
+---
+
+Pull tab/grid/card directives into external packages
```

Comment:
Should this be in here? Did this get picked up somehow accidentally?

Reply:
Hmm, yeah not sure how it got picked up here. This changeset was already resolved on main from this commit: https://github.com/executablebooks/mystjs/commit/037c8d5808077a1f34c4e37b8c4fecdf6ba59031

I'll delete the file from here...